code/mob/living/carbon/human.dm

```diff
@@ -3149,8 +3149,61 @@
 
 	if (head)
 		worn += head.name
-	else
-		worn += "hair"
+	else if (!src.mutantrace || istype(src.mutantrace, /datum/mutantrace/dwarf) || istype(src.mutantrace, /datum/mutantrace/flashy))
+		if ((src.bioHolder.mobAppearance.customization_first in hair_styles) || (src.bioHolder.mobAppearance.customization_second in hair_styles) || (src.bioHolder.mobAppearance.customization_third in hair_styles))
+			worn += "hair"
+		else
+			worn += "bald head"
+		if ((src.bioHolder.mobAppearance.customization_first in moustache_styles) || (src.bioHolder.mobAppearance.customization_second in moustache_styles) || (src.bioHolder.mobAppearance.customization_third in moustache_styles))
+			worn += "moustache"
+		if ((src.bioHolder.mobAppearance.customization_first in beard_styles) || (src.bioHolder.mobAppearance.customization_second in beard_styles) || (src.bioHolder.mobAppearance.customization_third in beard_styles))
+			worn += "beard"
+
+	if (src.mutantrace)
```

Comment:
I'm not really a fan of the if-else block inside this conditional. It looks like you're just adding a name for each individual mutantrace type. 

The "correct" way to write this code is to add a variable onto mutantrace and each mutantrace subtype will have that variable changed to fit. So you'd add that var in mutantrace.dm and then this if-statement could be replaced with something like: 
```
if (src.mutantrace)
    worn += src.mutantrace.your_variable_name
```

After that, I'd be happy to merge this.

Reply:
Thank you, I made and successfully tested the requested changes.